core/src/pagination/pagination.tsx

```diff
@@ -8,9 +8,21 @@ import { PaginationInput } from "./input/input";
 import s from "./pagination.module.css";
 
 export interface PaginationProps {
+	/**
+	 * The page to be selected. This will make this pagination controlled
+	 */
```

Comment:
The "controlled" part is a little redundant as we don't support uncontrolled pagination for now